addons/project/static/src/components/project_control_panel/project_control_panel.xml

```diff
@@ -2,15 +2,15 @@
 <templates xml:space="preserve">
 
     <t t-name="project.ProjectControlPanelContentBadge" owl="1">
-        <t t-tag="isProjectUser ? 'button' : 'span'" class="badge border d-flex p-2 ms-2 bg-view" data-hotkey="y">
+        <t t-tag="isProjectUser ? 'button' : 'span'" class="btn border badge d-flex p-2 align-items-center" data-hotkey="y">
             <span t-attf-class="o_status_bubble o_color_bubble_{{data.color}}"/>
             <span t-att-class="'fw-normal ms-1' + (data.color === 0 ? ' text-muted' : '')" t-esc="data.status"/>
         </t>
     </t>
 
     <t t-name="project.ProjectControlPanelContent" owl="1">
         <t t-if="showProjectUpdate">
-            <li t-if="isProjectUser" class="o_project_updates_breadcrumb ps-3" t-on-click="onStatusClick">
+            <li t-if="isProjectUser" class="o_project_updates_breadcrumb d-flex mb-lg-1" t-on-click="onStatusClick">
```

Comment:
I am not sure whether you additions are needed here. What motivated them ?

Reply:
> I am not sure whether you additions are needed here. What motivated them ?

We fixed alignment in small device. [link](https://tinyurl.com/292b4b58)

Reply:
I agree with setting the display to flex, but I am not sure about the bottom margin (the ```mb-lg-1``` class).

Reply:
If I am not mistaken though, the ```mb-lg-1``` class should only add bottom margin - which would not help separating the "New" and "Set" buttons. 
I tried removing it and the layout doesn't change. If it does for you, do send a screenshot of the result without it (cause it is surprising to me)

Reply:
> If I am not mistaken though, the `mb-lg-1` class should only add bottom margin - which would not help separating the "New" and "Set" buttons. I tried removing it and the layout doesn't change. If it does for you, do send a screenshot of the result without it (cause it is surprising to me)

Please check wihtout  bottom margin screen [link](https://www.awesomescreenshot.com/image/39126546?key=22eb8f01a3ccfc63280dd5834576bd56)

Reply:
Okay I see, thanks for pointing that out I was looking at the mobile view because of your previous text.